aws.yaml

```diff
@@ -1,653 +1,765 @@
+    version: 0.7.0
+  - name: cluster-operator
+    version: 2.0.2
+  - name: kubernetes
+    version: 1.15.5
+  - name: containerlinux
+    version: 2191.5.0
+  - name: coredns
+    version: 1.6.5
+  - name: calico
+    version: 3.9.1
+  - name: etcd
+    version: 3.3.15
+---
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  name: v10.1.1
+spec:
   state: deprecated
   date: 2020-01-10T08:00:00Z
-  authorities:
-    - name: app-operator
-      version: 1.0.0
-    - name: aws-operator
-      version: 8.0.2
-    - name: cert-operator
-      version: 0.1.0
-    - name: chart-operator
-      version: 0.7.0
-    - name: cluster-operator
-      provider: aws
-      version: 2.0.1
+  apps: []
   components:
-    - name: kubernetes
-      version: 1.15.5
-    - name: containerlinux
-      version: 2191.5.0
-    - name: coredns
-      version: 1.6.5
-    - name: calico
-      version: 3.9.1
-    - name: etcd
-      version: 3.3.15
-- version: 10.1.0
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-operator
+    version: 8.0.2
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    version: 2.0.1
+  - name: kubernetes
+    version: 1.15.5
+  - name: containerlinux
+    version: 2191.5.0
+  - name: coredns
+    version: 1.6.5
+  - name: calico
+    version: 3.9.1
+  - name: etcd
+    version: 3.3.15
+---
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  name: v10.1.0
+spec:
   state: deprecated
   date: 2019-12-18T14:00:00Z
-  authorities:
-    - name: app-operator
-      version: 1.0.0
-    - name: aws-operator
-      version: 8.0.2
-    - name: cert-operator
-      version: 0.1.0
-    - name: chart-operator
-      version: 0.7.0
-    - name: cluster-operator
-      provider: aws
-      version: 2.0.0
+  apps: []
   components:
-    - name: kubernetes
-      version: 1.15.5
-    - name: containerlinux
-      version: 2191.5.0
-    - name: coredns
-      version: 1.6.5
-    - name: calico
-      version: 3.9.1
-    - name: etcd
-      version: 3.3.15
-- version: 9.2.2
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-operator
+    version: 8.0.2
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    version: 2.0.0
+  - name: kubernetes
+    version: 1.15.5
+  - name: containerlinux
+    version: 2191.5.0
+  - name: coredns
+    version: 1.6.5
+  - name: calico
+    version: 3.9.1
+  - name: etcd
+    version: 3.3.15
+---
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
```

Comment:
This is nice, What do you think to put a release note at the annotation?

Reply:
I like the idea - I will add it in a follow up because we need to talk to sig UX first.